styleguide/components/31-molecules/pagination/pagination.config.js

```diff
@@ -6,36 +6,36 @@ module.exports = {
   name: 'pagination',
```

Comment:
Perhaps a little inline documentation for the numbers in this file would be in order? :P